README.md

```diff
@@ -0,0 +1 @@
+pdbvangogh is a package intended to enable style transfer on molecular structures with artistic backgrounds
```

Comment:
Typically you'd add a top-level LICENSE file at the root level w/whatever you end up going with (MIT, Apache 2.0, etc.)